modules/apps/journal/journal-service/src/main/java/com/liferay/journal/internal/transformer/DLPreviewURLRefreshTimestampTransformerListener.java

```diff
@@ -0,0 +1,155 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.journal.internal.transformer;
+
+import com.liferay.document.library.kernel.exception.NoSuchFileEntryException;
+import com.liferay.document.library.kernel.service.DLAppLocalService;
+import com.liferay.journal.constants.JournalPortletKeys;
+import com.liferay.petra.string.StringBundler;
+import com.liferay.portal.kernel.exception.PortalException;
+import com.liferay.portal.kernel.log.Log;
+import com.liferay.portal.kernel.log.LogFactoryUtil;
+import com.liferay.portal.kernel.repository.model.FileEntry;
+import com.liferay.portal.kernel.templateparser.BaseTransformerListener;
+import com.liferay.portal.kernel.templateparser.TransformerListener;
+import com.liferay.portal.kernel.util.Validator;
+
+import java.util.Date;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+
+/**
+ * @author István András Dézsi
+ */
+@Component(
+	immediate = true,
+	property = "javax.portlet.name=" + JournalPortletKeys.JOURNAL,
+	service = TransformerListener.class
+)
+public class DLPreviewURLRefreshTimestampTransformerListener
+	extends BaseTransformerListener {
+
+	@Override
+	public String onOutput(
+		String output, String languageId, Map<String, String> tokens) {
+
+		if (_log.isDebugEnabled()) {
```

Comment:
Can you remove this debug message? alternatively, use a more detailed message.

Reply:
This is the same debug message that is used in the "neighbour" classes `ViewCounterTransformerListener`, `LocaleTransformerListener`, `TokensTransformerListener` etc. as well. Admittedly, they are quite old code, and if you insist, I'll remove the debug statement.